src/App.tsx

```diff
@@ -1,25 +1,52 @@
-import React, { useState } from 'react';
+import React, { useCallback, useEffect, useState } from 'react';
 import './App.scss';
 import './styles/general.scss';
 import { TodoList } from './components/TodoList';
 import { CurrentUser } from './components/CurrentUser';
+import { TodoType } from './types/TodoType';
+import { getTodosFromServer } from './api/api';
 
 const App: React.FC = () => {
+
   const [
     selectedUserId,
-    // setSelectedUserId,
-  ] = useState(0);
+    setSelectedUserId,
+  ] = useState<number | null>(0);
```

Comment:
don't use null for selectedUserId
```suggestion
  ] = useState(0);
```